README.md

```diff
@@ -63,30 +70,16 @@ The application will be available at http://localhost:3000
 The application includes several Python components for PDF processing:
 
 - Site Crawler: Discovers PDF files on government websites
```

Comment:
"Discovers PDF files ... " -> "Downloads PDF files and their metadata from government websites"

Just trying to capture that this step has absorbed the "Metadata Downloader" below.